From ad8a8629c58215d975dc0c75495d6bcd6b594548 Mon Sep 17 00:00:00 2001
From: Timo Juhani Lindfors <timo.lindfors@iki.fi>
Date: Thu, 28 Feb 2013 12:58:37 +0200
Subject: [PATCH] manpager: fix warnings by adding breaks between SEE ALSO
 references

This patch fixes the following warnings from man:

usr/share/man/man3/tapset::ansi.3stap.gz 99: warning [p 2, 1.8i]: cannot adjust line
usr/share/man/man3/tapset::context-unwind.3stap.gz 48: warning [p 1, 7.0i]: cannot adjust line
usr/share/man/man3/tapset::conversions-guru.3stap.gz 67: warning [p 1, 8.8i]: cannot adjust line
usr/share/man/man3/tapset::conversions.3stap.gz 109: warning [p 2, 3.5i]: cannot adjust line
usr/share/man/man3/tapset::inet_sock.3stap.gz 27: warning [p 1, 3.8i]: cannot adjust line
usr/share/man/man3/tapset::ioblock.3stap.gz 53: warning [p 1, 7.3i]: cannot adjust line
usr/share/man/man3/tapset::ioscheduler.3stap.gz 118: warning [p 2, 4.8i]: cannot adjust line
usr/share/man/man3/tapset::ipmib.3stap.gz 139: warning [p 2, 6.8i]: cannot adjust line
usr/share/man/man3/tapset::irq.3stap.gz 77: warning [p 1, 10.0i]: cannot adjust line
usr/share/man/man3/tapset::kprocess.3stap.gz 64: warning [p 1, 9.0i]: cannot adjust line
usr/share/man/man3/tapset::linuxmib.3stap.gz 43: warning [p 1, 5.8i]: cannot adjust line
usr/share/man/man3/tapset::memory.3stap.gz 142: warning [p 2, 7.7i]: cannot adjust line
usr/share/man/man3/tapset::netfilter.3stap.gz 75: warning [p 1, 10.7i]: cannot adjust line
usr/share/man/man3/tapset::networking.3stap.gz 134: warning [p 2, 6.2i]: cannot adjust line
usr/share/man/man3/tapset::nfs.3stap.gz 191: warning [p 3, 1.8i]: cannot adjust line
usr/share/man/man3/tapset::nfs_proc.3stap.gz 143: warning [p 2, 7.2i]: cannot adjust line
usr/share/man/man3/tapset::proc_mem.3stap.gz 139: warning [p 2, 7.3i]: cannot adjust line
usr/share/man/man3/tapset::queue_stats.3stap.gz 105: warning [p 2, 2.3i]: cannot adjust line
usr/share/man/man3/tapset::rpc.3stap.gz 163: warning [p 2, 9.8i]: cannot adjust line
usr/share/man/man3/tapset::scheduler.3stap.gz 143: warning [p 2, 7.2i]: cannot adjust line
usr/share/man/man3/tapset::signal.3stap.gz 290: warning [p 4, 4.2i]: cannot adjust line
usr/share/man/man3/tapset::socket.3stap.gz 206: warning [p 3, 5.2i]: cannot adjust line
usr/share/man/man3/tapset::stap_staticmarkers.3stap.gz 243: warning [p 3, 8.2i]: cannot adjust line
usr/share/man/man3/tapset::stopwatch.3stap.gz 67: warning [p 1, 8.8i]: cannot adjust line
usr/share/man/man3/tapset::target_set.3stap.gz 27: warning [p 1, 3.8i]: cannot adjust line
usr/share/man/man3/tapset::task.3stap.gz 161: warning [p 2, 9.0i]: cannot adjust line
usr/share/man/man3/tapset::task_time.3stap.gz 98: warning [p 2, 2.2i]: cannot adjust line
usr/share/man/man3/tapset::tcp.3stap.gz 90: warning [p 2, 0.8i]: cannot adjust line
usr/share/man/man3/tapset::tcpmib.3stap.gz 123: warning [p 2, 4.8i]: cannot adjust line
usr/share/man/man3/tapset::timestamp_gtod.3stap.gz 43: warning [p 1, 5.8i]: cannot adjust line
usr/share/man/man3/tapset::timestamp_monotonic.3stap.gz 79: warning [p 1, 10.2i]: cannot adjust line
usr/share/man/man3/tapset::ucontext-unwind.3stap.gz 35: warning [p 1, 5.0i]: cannot adjust line
usr/share/man/man3/tapset::uconversions.3stap.gz 253: warning [p 3, 10.7i]: cannot adjust line
usr/share/man/man3/tapset::udp.3stap.gz 66: warning [p 1, 9.3i]: cannot adjust line
---
 doc/SystemTap_Tapset_Reference/manpager |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/doc/SystemTap_Tapset_Reference/manpager b/doc/SystemTap_Tapset_Reference/manpager
index a250162..1dab7aa 100755
--- a/doc/SystemTap_Tapset_Reference/manpager
+++ b/doc/SystemTap_Tapset_Reference/manpager
@@ -89,11 +89,14 @@ for i in `cat ../manpageus` ; do
   echo "" >> $i.footer
   echo ".SH SEE ALSO" >> $i.footer
   for j in `cat $i.funcnames` ; do
+    echo ".BR" >> $i.footer
     echo ".IR \\%function::"$j" (3stap)," >> $i.footer
   done
   for j in `cat $i.probenames` ; do
+    echo ".BR" >> $i.footer
     echo ".IR \\%probe::"$j" (3stap)," >> $i.footer
   done
+  echo ".BR" >> $i.footer
   echo ".IR \\%stap (1)," >> $i.footer
   echo ".IR \\%stapprobes (3stap)" >> $i.footer
 done
-- 
1.7.10.4

